src/test/graph_export_test.cpp

```diff
@@ -175,15 +329,23 @@ TEST_F(GraphCreationTest, positivePluginConfigAll) {
 }
 
 TEST_F(GraphCreationTest, positivePluginConfigOne) {
-    ovms::GraphSettingsImpl graphSettings;
-    graphSettings.pluginConfig.kvCachePrecision = "u8";
+    ovms::HFSettingsImpl hfSettings;
+    hfSettings.graphSettings.pluginConfig.kvCachePrecision = "u8";
 
     std::string graphPath = ovms::FileSystem::appendSlash(this->directoryPath) + "graph.pbtxt";
     std::unique_ptr<ovms::GraphExport> graphExporter = std::make_unique<ovms::GraphExport>();
-    auto status = graphExporter->createGraphFile(this->directoryPath, graphSettings);
+    auto status = graphExporter->createGraphFile(this->directoryPath, hfSettings);
     ASSERT_EQ(status, ovms::StatusCode::OK);
 
     std::string graphContents = GetFileContents(graphPath);
     std::cout << graphContents << std::endl;
     ASSERT_EQ(expectedOneSettingPluginGraphContents, graphContents);
 }
+
+TEST_F(GraphCreationTest, negativeCreateFile) {
```

Comment:
All graph parameters could contain special chars that would break either pbtxt or config.json. Validation of those is part of separate task?